test/reducedim.jl

```diff
@@ -209,9 +209,9 @@ end
 
     for f in (minimum, maximum)
         @test_throws "reducing over an empty collection is not allowed" f(A, dims=1)
-        @test isequal(f(A, dims=2), zeros(Int, 0, 1))
+        @test_broken isequal(f(A, dims=2), zeros(Int, 0, 1))
```

Comment:
I marked these as broken as a reminder to myself; I'm not entirely sure that we want/need to preserve these behaviors.

Reply:
This is now split out into #55628.